__tests__/app.test.js

```diff
@@ -23,3 +24,18 @@ describe("Get topics", () => {
       });
   });
 });
+
+describe("Get API", () => {
+  test("API request should return the object from endpoints.json", () => {
+    return request(app)
+      .get("/api/")
+      .expect(200)
+      .then((result) => {
+        let endPointKeys = Object.keys(endPoint);
+        expect(endPointKeys).toEqual(Object.keys(result.body));
```

Comment:
This can just be expect(endPoint).toEqual(result.body).

Reply:
Thank you :)